packages/design-system/package.json

```diff
@@ -34,16 +34,15 @@
     "body-scroll-lock": "^3.1.5",
     "polished": "^4.1.0",
     "prop-types": "^15.7.2",
-    "react-is": "^17.0.2",
     "react-modal": "^3.12.1",
     "react-spring": "^9.4.3",
     "react-tabs": "^3.2.2",
-    "react-toast-notifications": "^2.4.3",
-    "styled-components": "^5.2.1"
+    "react-toast-notifications": "^2.4.3"
   },
   "peerDependencies": {
     "react": "^17.0.0",
-    "react-dom": "^17.0.0"
+    "react-dom": "^17.0.0",
```

Comment:
should we also bump these to 18.x to match the dashboards repo? or maybe expand to e.g. `>17.0.0` so it still supports legacy use cases?

Reply:
Changed it to >17 because the types for the package at the moment don't work with 18, so technically requiring `^18` is a problem

Reply:
right i forgot you said that